src/renderer-backend-egl.cpp

```diff
@@ -170,6 +177,14 @@ struct wpe_renderer_backend_egl_target_interface fdo_renderer_backend_egl_target
         auto& target = *reinterpret_cast<Target*>(data);
         target.m_impl->frameRendered();
     },
+#if WPE_CHECK_VERSION(1,9,1)
+    // deinitialize
+    [](void* data)
+    {
+        auto& target = *reinterpret_cast<Target*>(data);
+        target.m_impl->deinitialize();
+    },
+#endif
```

Comment:
The `WPE_CHECK_VERSION` conditional here means that when building with an older libwpe the `::deinitialize()` method will never get called, and we would leak resources.

Wouldn't it make sense to only compile the whole DMA-BUF pool support when building againsy  libwpe≥1.9.1?